workspace_tools/targets.py

```diff
@@ -1269,18 +1269,21 @@ def __init__(self):
 class RBLAB_BLENANO(MCU_NRF51_16K):
     def __init__(self):
         MCU_NRF51_16K.__init__(self)
+        self.supported_toolchains = ["GCC_ARM"]
```

Comment:
supported toolchains are for compiling (basically means there is cmsis support for this toolchain). This target inherits from nrf51 base class which provides supported toolchains.

Exporting is done as you did in the export scripts. Thus these changes here could be removed .